embedchain/llm/octoai.py

```diff
@@ -0,0 +1,59 @@
+import os
+from typing import Optional
+
+from langchain.llms.octoai_endpoint import OctoAIEndpoint
+
+from embedchain.config import BaseLlmConfig
+from embedchain.helper.json_serializable import register_deserializable
+from embedchain.llm.base import BaseLlm
+
+@register_deserializable
+class OctoaiLlm(BaseLlm):
+    def __init__(self, config: Optional[BaseLlmConfig] = None):
+        if "OCTOAI_API_TOKEN" not in os.environ:
+            raise ValueError("Please set the OCTOAI_API_TOKEN environment variable.")
+        if "ENDPOINT_URL" not in os.environ:
+            raise ValueError ("Please enter your model's endpoint url")
```

Comment:
I think maybe `Please set the OCTOAI_ENDPOINT_URL environment variable` sounds more helpful to the users. Please let me know what do you think

Reply:
Makes more sense. I fixed it